scripts/update-creatives.js

```diff
@@ -89,9 +107,28 @@ function handleError(err) {
   console.log('because', err.stack);
 }
 
-Bluebird.resolve(query)
+function splitBatches(lineItems){
+  var batches = _.chunk(lineItems, 400);
+  progressBar = new ProgressBar('Progress [:bar] :percent :elapseds', {
+    total: batches.length + 1
+  });
+  return batches;
+}
+
+function advanceProgress(){
+  progressBar.tick();
+}
+
+// this function is to help debugging
+function log(x){
+  console.log(x);
+}
```

Comment:
I found myself using this over and over when debugging, so I thought it might be useful to include.